status/reporter_gitlab_test.go

```diff
@@ -218,6 +218,32 @@ var _ = Describe("GitLabReporter", func() {
 				})).To(Succeed())
 		})
 
+		It("creates a commit status for push snapshot with correct textual data without comments", func() {
+
+			// Removing the pull request annotation and adding the push label
+			delete(hasSnapshot.Annotations, gitops.PipelineAsCodePullRequestAnnotation)
+			hasSnapshot.Annotations[gitops.PipelineAsCodeEventTypeLabel] = "Push"
```

Comment:
To prevent possible conflicts in the future, maybe we could create a deepcopy of `hasSnapshot` and store it in a different variable, e.g.
```
pushSnapshot := hasSnapshot.DeepCopy()
delete(pushSnapshot.Annotations, gitops.PipelineAsCodePullRequestAnnotation)
pushSnapshot.Annotations[gitops.PipelineAsCodeEventTypeLabel] = "Push"
```

Reply:
Thanks Kruno willdo!